src/communication/communicationmanager.cpp

```diff
@@ -0,0 +1,85 @@
+#include "communicationmanager.hpp"
+#include "openchannel.hpp"
+#include "securechannel.hpp"
+
+aos::Error CommunicationManager::Init(CommChannelItf& transportChannel)
```

Comment:
Let's have two interfaces to make code more readable:

transport (like vchan, pipe) - TransportItf with Read, Write methods
communication channels  - CommChannelItf with Read, Write methods